### DISCUSSION TITLE
Unable to update Nginx Proxy Manager: python3 error while trying to install certbot-dns-cloudflare

### DISCUSSION THREAD

**Discussion by ajtatum** [tteck/Proxmox#4016] [Help Needed] [answered]
This is the first time I've had an issue updating NPM. I have to LXCs and they're both giving the same error.

```
 ✓ Stopped Services
 ✓ Cleaned Old Files
 ✓ Downloaded NPM v2.12.1
 | Setting up Enviroment   
[ERROR] in line 137: exit code 0: while executing command python3 -m pip install --no-cache-dir certbot-dns-cloudflare &> /dev/null
```

Naturally NPM doesn't work afterwards and I have to revert from a backup. I'm not sure why it's suddenly having issues now.

Any help would be appreciated!

**Comment by EnduringGuerila** [comment]
Same here!

**Reply by tteck** [reply, marked as answer]
fixed https://github.com/tteck/Proxmox/commit/38d3f2c868bb426895aeb59ea41e89acd465b06a

**Comment by LucasJanin** [comment]
Same issue on my side too.
